src/styles/inc/_vars.scss

```diff
@@ -0,0 +1,43 @@
+@use "sass:math" as *;
+@use "sass:list" as *;
+
+@function rem($value) {
+  $remvalue: div($value, $mainfont) + rem;
+
+  @return $remvalue;
+}
+
+$white: #fff;
+$black: #000;
+$calendardays: 31;
+$mainfont: 16;
+$weekdays: (
+  0: "mon",
+  1: "tue",
+  2: "wed",
+  3: "thu",
+  4: "fri",
+  5: "sat",
+  6: "sun",
+);
+$gap: 1px;
+$daywidth: 100px;
+
+// <Mixins-media-queries>=========================================================================================================
```

Comment:
Remove comment. Try to write obvious code with meaningful variable and function naming
Or move this logic to separate file